### ISSUE TITLE
Add mysql/mariadb mix-and-match odbc projects

### ISSUE THREAD

**Issue by SwooshyCueb** [irods/irods_testing_environment#208]
Prerequisite: #167

Presently, iRODS works just fine with MariaDB, provided a MySQL ODBC driver is used.
Let's add projects for testing iRODS with mis-matched MySQL/MariaDB ODBC drivers.

**Comment by alanking** [comment]
One effort which could make this slightly easier to do would be https://github.com/irods/irods_testing_environment/issues/58, which is basically a stand-in issue for re-doing the way we set up the ODBC drivers for each platform. Right now, I would say that it is less than ideal to have to add a function for each project.

That being said, it's not a prerequisite. Just mentioning it in case it could help to know that.

**Comment by SwooshyCueb** [comment]
> I would say that it is less than ideal to have to add a function for each project.

I agree completely. The less project-specific stuff we have outside the projects folder, the better.

**Comment by korydraughn** [comment]
As noted in the linked issue, #223, I've confirmed that all unit/core tests pass when using the MySQL 8 ODBC drivers with MariaDB.
